notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1139 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ! gcloud ai models delete $CUSTOM_JOB_MODEL --quiet
</pre><p>Prefer Vertex SDK deletion method instead of gcloud.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
Done.